core/src/main/java/org/apache/accumulo/core/conf/cluster/ClusterConfigParser.java

```diff
@@ -44,12 +44,11 @@
 
 public class ClusterConfigParser {
 
-  private static final Pattern GROUP_NAME_PATTERN =
-      Pattern.compile("^[a-zA-Z_]{1,}[a-zA-Z0-9_]{0,}$");
+  private static final Pattern GROUP_NAME_PATTERN = Pattern.compile("^[a-zA-Z_]+[a-zA-Z0-9_]*$");
```

Comment:
Original changes looked good. Moved the underscore match so that a resource group name of `_` is not considered valid

```suggestion
  private static final Pattern GROUP_NAME_PATTERN = Pattern.compile("^([a-zA-Z]_?)+[a-zA-Z0-9_]*$");
```